### PR TITLE
New CHWS and CDWS, abstracts, fields, subfields

### PR DESCRIPTION
**Pull request by evgeniya-engel**

### CODE REVIEW

**Comment by evgeniya-engel** [comment]
@cstirdivant this one is ready for your review. Thank you!

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 1]
```diff
@@ -1269,6 +1287,9 @@ literals:
 - water_volume_accumulator:
     flexible_min: -0.000315451
     flexible_max: 0.315451
+- makeup_water_volume_accumulator:
+    flexible_min: -0.000315451
+    flexible_max: 0.315451
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:1292]: This is not a reasonable value range.

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/fields/telemetry_fields.yaml (hunk 1 above)
Comment (evgeniya-engel) [reply ontology/yaml/resources/fields/telemetry_fields.yaml:1292]: Would {min: 0, max: 10e6} be more adequate? Max value for water_volume_accumulator in our historical data is 6e6.

Comment (cstirdivant) [reply ontology/yaml/resources/fields/telemetry_fields.yaml:1292]: what units? the units required here are cubic meters

Comment (evgeniya-engel) [reply ontology/yaml/resources/fields/telemetry_fields.yaml:1292]: The reference data is in cubic meters. Actually 10e6 looks too large for cubic meters so I took 98th percentile instead and it's 95000 cubic meters. How about {min: 0, max: 1e5}?

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**